src/net.cpp

```diff
@@ -1122,6 +1121,7 @@ bool CConnman::AttemptToEvictConnection()
     LOCK(cs_vNodes);
     for (CNode* pnode : vNodes) {
         if (pnode->GetId() == *node_id_to_evict) {
+            LogPrint(BCLog::NET, "selected %s connection for eviction peer=%d; disconnecting\n", pnode->ConnectionTypeAsString(), pnode->GetId());
```

Comment:
20724: should probably use `NET_NETCONN` here like we do in similar cases to be able to log these with `netconn` category which helps getting connection specific output only without all the `net` spam.

Reply:
Resolved in latest push